src/Tools/FormExporter.php

```diff
@@ -206,6 +211,23 @@ protected function extractDataRows(array $resultsArray)
                 }
                 $this->extractColumnNames($resultRow);
                 $data = json_decode($resultRow[$this->dataFieldName], true);
+
+                // Created Date
+                $createdDate = null;
+                try {
+
+                    $createdDate = new \DateTime($resultRow['created']);
+
+                } catch (\Exception $e) {
+                    throw new \InvalidArgumentException("createdDate '{$createdDate}' was unable to be processed.");
```

Comment:
You should make this {$resultRow['created']} as $createdDate would likely be null here.